app.js

```diff
@@ -0,0 +1,68 @@
+const express = require("express");
+const cors = require("cors");
+// const axios = require("axios");
+const body  = require("body-parser");
+const mySchool = require('./School')
+
+const port = 3000;
+
+const app = express();
+app.use(cors());
+app.use(body.urlencoded({extended: false}));
+
+// let mySchool = new School();
+
+const getAllClasses = (req, res) => {
+    console.log("these are the classes" + mySchool.classes)
+    res.json({
+        allClasses: mySchool.classes
+        
+    })
+}
+
+const checkIfClassExists = (req, res, next) =>  {
+        
+    if (req.class) {
+        //here logic for if input != req.body.class then add the class
+        res.status(200).json({//chain it to send back a json and let it know it's all good
+        message: "Success your class has been added!",
+        data: req.class
```

Comment:
You wouldn't want to return your request inside of your response. You might be creating an illusion that everything went through and was successful even if it wasn't. 
Also this function is supposed to only check if the class exists. If it does, you need to send an error message, if it doesn't you need to move on to the `next()` function - which would be to actually add the class.